### PR TITLE
hlaminer 1.3.1

### PR DESCRIPTION
**Pull request by sjackman**
The previous commit was mistakenly tagged as version 1.3.0,
although it was in fact version 1.3.1.

### CODE REVIEW

**Comment by jonchang** [comment]
Thanks for the PR!

**Comment by jonchang** [comment]
@sjackman :penguin:

File: hlaminer.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class Hlaminer < Formula
   desc "HLA predictions from next-generation shotgun (NGS) sequence read data"
   homepage "http://www.bcgsc.ca/platform/bioinfo/software/hlaminer"
   url "http://www.bcgsc.ca/platform/bioinfo/software/hlaminer/releases/1.3.1/HLAminer_v1-3-1.tar.gz"
-  version "1.3.0"
+  version "1.3.1"
   sha256 "a009575d80484c7c1dd188f29c21419ccca3ff7feea31128d0906cfc944959fe"
```
Comment (warrenlr) [inline hlaminer.rb]: sha256 appears wrong. Could you update to:
91d510fe9d5c731e6756f6680738bc64fb61844758ede2a4160608945b6ef8cf

Comment (sjackman) [inline hlaminer.rb]: I've opened PR 4713 to correct the SHA.
Did the contents of this file change?

**Pull request closed without merging**